### PR TITLE
[11.0][ADD] sale_stock_picking_note: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module extend sale order to include a note that will be set as the picking
note.

cc @Tecnativa

### CODE REVIEW

File: sale_stock_picking_note/models/sale_stock.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    def _get_new_picking_values(self):
+        vals = super()._get_new_picking_values()
+        sale_note = self.sale_line_id.order_id.picking_note
+        if sale_note:
+            vals['note'] = sale_note
```
Comment (nikul-serpentcs) [inline sale_stock_picking_note/models/sale_stock.py:22]: @chienandalu IMO here use `vals.get('note')` and check in If condition `if sale_note and vals.get('note'):`

**Review by nikul-serpentcs — COMMENTED** [review]
Small Changes

File: sale_stock_picking_note/models/sale_stock.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_stock_picking_note/models/sale_stock.py:22]: I have put a better formula IMO

**Review by pedrobaeza — APPROVED** [review]

**Review by tafaRU — COMMENTED** [review]
Code review and test
Would it be possible to add some unit tests ?

**Comment by rafaelbn** [comment]
@tafaRU 

>  Would it be possible to add some unit tests ?

```
@codecov
codecov/patch — 100% of diff hit (target 92.84%)
Details
@codecov
codecov/project — 92.94% (+0.09%) compared to d5e0b1b 
```

:smile: :+1:

**Review by DaoTheKien — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

@pedrobaeza @tafaRU Ready to merge

**Pull request merged**